native_client/javascript/index.js

```diff
@@ -111,55 +123,56 @@ Model.prototype.createStream = function() {
     return ctx;
 }
 
+function Stream(nativeStream) {
```

Comment:
Pretty sure you'll need to add `Stream` to `doc/NodeJS-API.rst` like we have `Model`